engine/src/cython/initialize.cpp

```diff
@@ -62,7 +62,12 @@ std::string get_ip(const std::string & iface_name = "eth0") {
 }
 
 // simple_log: true (no timestamp or log level)
-void create_logger(std::string fileName, std::string loggingName, int ralId, std::string flush_level, bool simple_log=true){
+void create_logger(std::string fileName,
+	std::string loggingName,
+	int ralId, std::string flush_level,
+	std::map<std::string, std::string> config_options,
```

Comment:
Here please pass in the logging level and not the config_options

Reply:
ok. Updated.